src/main/java/org/blockserver/net/protocol/pe/login/RaknetUnconnectedPong.java

```diff
@@ -7,7 +7,11 @@
 public class RaknetUnconnectedPong implements PeProtocolConst{
 	private ByteBuffer bb;
 	public RaknetUnconnectedPong(long ping, long server, byte[] magic, String name){
-		name = "MCCPP;Demo;" + name;
+		int protocol_version = 27; //Minecraft PE Protocol Version
```

Comment:
Oh. I just threw that together in a minute or two. No reason as to why (or why not) I did it the way I did.